### ISSUE TITLE
"from import as" formatting setting?

### ISSUE THREAD

**Issue by Modexus** [astral-sh/ruff#9562]
Is there a way to control formatting of "from import as" blocks" such as:
```python
from foo import (
    bar as baz,
    bar2 as baz2,
    bar3 as baz3,
    bar4 as baz4,
    bar5 as baz5,
    bar6 as baz6,
)
```

this currently formats to
```python
from foo import (
    bar as baz,
)
from foo import (
    bar2 as baz2,
)
from foo import (
    bar3 as baz3,
)
from foo import (
    bar4 as baz4,
)
from foo import (
    bar5 as baz5,
)
from foo import (
    bar6 as baz6,
)
```
As I have a bunch of these it makes it quite unreadable and there does not seem to be a way to ignore it without completely disabling I001.

**Comment by charliermarsh** [comment]
Example here (notice that it's not flagged as a violation): https://play.ruff.rs/15ce8c9b-7e61-4831-8d63-24af163b5a0e

**Comment by Modexus** [comment]
Thanks a lot! I missed that isort is configurable in the settings, very useful.

**Comment by charliermarsh** [comment]
No prob! I think at one point we defaulted this setting to true, but changed it to false to match isort’s defaults.
